modules/fw/manifests/networks.pp

```diff
@@ -36,18 +36,10 @@
     # AWS Networks; See https://github.com/mozilla-releng/build-cloud-tools/blob/master/configs/subnets.yml
 
     # USE1 Network CIDRs
-    $use1_test    = [ '10.134.40.0/21', '10.134.56.0/22', '10.134.60.0/22', '10.134.156.0/22' ]
-    $use1_build   = [ '10.134.52.0/22', '10.134.164.0/24' ]
-    $use1_srv     = [ '10.134.48.0/22' ]
```

Comment:
srv has typically been were we put infra like puppet masters and log aggregators.  I assume it should stick around.

Reply:
Agreed, and we put non-signing scriptworkers in there too for use1, but this definition doesn't seem to be used anywhere else in puppet. Would you prefer to leave it as a reference ?

Reply:
Yes.  I'd prefer to keep the reference even if it isn't used.